Deployment/default.ps1

```diff
@@ -0,0 +1,62 @@
+Properties {
+	$Deployment = Split-Path $psake.build_script_file
+	$sln = "$Deployment\..\Solutions\ScriptRunner.sln"
+	$Artifacts = "$Deployment\Artifacts"
+	$Tools = "$Deployment\Tools"
+	$TestResult_xml = "$Deployment\TestResult.xml"
+	$BuildTools = "$Tools\BuildTools"
+	$nuget_exe = "$Tools\nuget.exe"
+	$packages_config = "$Deployment\packages.config"
+	$build_xml = "$Deployment\build.xml"
+	$nunit_exe = "$BuildTools\NUnit.ConsoleRunner.3.8.0\tools\nunit3-console.exe"
+	$rapbuilder_exe = "$BuildTools\RelativityDev.RapBuilder.0.0.0.3-alpha\lib\kCura.RAPBuilder.exe"
+	$Packages = "$Deployment\Packages"
+	$scriptrunner_rap = "$Deployment\ScriptRunner.rap"
+
+	$Version = $null
+}
+
+Task Default -Depends UnitTest
+
+Task PackageBuild -Depends RestorePackages {
+	Exec { 
+		msbuild "$sln" `
```

Comment:
Surprised you don't need to specify a publish profile for the custom page

Reply:
Yeah. I was surprised too but the artifacts are contained in the output in a _PublishedWebsites folder. I think there is some setting on one of the csproj files that takes care of everything.